apps/wallet-dashboard/app/(protected)/assets/page.tsx

```diff
@@ -23,38 +21,59 @@ const ASSET_CATEGORIES: { label: string; value: AssetCategory }[] = [
         label: 'Other',
         value: AssetCategory.Other,
     },
+    {
```

Comment:
I don't think it's planned to display hidden assets on the dashboard :see_no_evil:

Reply:
correct, the logic to display hidden assets should be removed from this file since the dashboard doesnt have the feature implemented